libs/mat-table-filter/src/lib/mat-table-filter-header.ts

```diff
@@ -27,6 +28,7 @@ import {MatTableHeaderType} from './models/header-type';
 export class MatTableFilterHeader implements AfterViewInit {
   @Input({required: false}) matTableFilterHeaderType: MatTableHeaderType =
     'string';
+  @Input({required: false}) isHeaderCaseSensitive: CaseSensitivityType = true;
```

Comment:
The naming is important here. To infer the developer that this field is coming from `MatTableFilterHeader`, it should prefixed as `matTableFilter`. So changing the attribute name to `matTableFilterCaseSensitive` or `matTableFilterIsCaseSensitive` would be much better.